### PR TITLE
Bug 1898806 - Move Preview URL context to an info button

### PR DESCRIPTION
**Pull request by sarahhjchung**
[**Bug 1898806**](https://bugzilla.mozilla.org/show_bug.cgi?id=1898806)

Changes made:
- Created a new `InfoPopover` component using [shadcn popover](https://ui.shadcn.com/docs/components/popover) 
- Added a new label for the last column in the message and experiment tables called "Visuals" and moved the Preview URL context from the top of Skylight into the info button inside that label
- Fixed a `fillOpacity` bug
- Renamed the `InfoIcon` tooltip button to `InfoTooltip` 
- Added `aria-label`s to the info and toggle buttons to improve accessibility scores

### CODE REVIEW

File: __tests__/components/ui/infotooltip.test.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { fireEvent, render, screen } from "@testing-library/react";
```
Comment (sarahhjchung) [inline __tests__/components/ui/infotooltip.test.tsx:1]: This file is just the original `infoicon.test.tsx` file renamed to `infotooltip.test.tsx`

File: app/columns.tsx [hunk 2]
```diff
@@ -94,6 +88,21 @@ export type BranchInfo = {
...
 
+const previewURLInfoButton = (
+  <InfoPopover
+    iconSize={14}
+    content={
+      <p>
+        To make the Preview URLs work, load <code>about:config</code> in Firefox
+        and set{" "}
+        <code>browser.newtabpage.activity-stream.asrouter.devtoolsEnabled</code>{" "}
+        to true; a Firefox 126 build is required.
```
Comment (dmose) [inline app/columns.tsx:99]:
```suggestion
        to true; Firefox 126 or newer is required.
```

File: app/columns.tsx [hunk 3]
```diff
@@ -182,6 +197,7 @@ export const experimentColumns: ColumnDef<RecipeOrBranchInfo>[] = [
                     style: { cursor: 'pointer' },
                   }}
                   data-testid="toggleBranchRowsButton"
+                  aria-label="Toggle Branches"
```
Comment (dmose) [inline app/columns.tsx:200]: Good catch!

File: app/page.tsx [hunk 4]
```diff
@@ -4,7 +4,7 @@ import { getDashboard, getDisplayNameForTemplate, getTemplateFromMessage, _isAbo
 import { NimbusRecipeCollection } from "../lib/nimbusRecipeCollection"
 import { _substituteLocalizations } from "../lib/experimentUtils.ts";
 
-import { InfoIcon } from "@/components/ui/infoicon.tsx";
+import { InfoTooltip } from "@/components/ui/infotooltip.tsx";
```
Comment (dmose) [inline app/page.tsx:7]: I like the renaming; it adds clarity.

**Review by dmose — APPROVED** [review]
This looks great; thanks so much!  r=dmose, with one wording nit fix.

**Pull request merged**